cvat-canvas/src/typescript/editHandler.ts

```diff
@@ -380,6 +370,19 @@ export class EditHandlerImpl implements EditHandler {
     }
 
     private closeEditing(): void {
+        if (this.isEditing) {
+            const { offset } = this.geometry;
+            const head = this.editedShape.attr('points').split(' ').slice(0, this.editData.pointID).join(' ');
+            const stringifiedPoints = `${head} ${this.editLine.node.getAttribute('points').slice(0, -2)}`;
+            const points = pointsToNumberArray(stringifiedPoints)
+                .slice(0, -2)
+                .map((coord: number): number => coord - offset);
+            if (points.length >= 2 * 2) { // minimumPoints * 2
+                const { state } = this.editData;
+                this.onEditDone(state, points);
+            }
+        }
+        this.isEditing = false;
         this.release();
```

Comment:
Since the polygon editing was affected, i am thinking to add checktype for shape, something like this
`
private closeEditing(): void {
        if (this.isEditing && this.editData.state.shapeType === 'polyline') {
           ...
        } else {
            this.onEditDone(null, null);
        }
        this.isEditing = false;
        this.release();
    }
`
So I think I shall keep it outside.
I am open to suggestions.